### PR TITLE
Some very minor copy updates.

### PR DESCRIPTION
**Pull request by nathenharvey**
Signed-off-by: Nathen Harvey <nharvey@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/demo/steps/8.html.slim [hunk 1]
```diff
@@ -21,7 +21,7 @@ classes: demo_step
         .columns.large-12
           h1.h4="Step 8: It's Build Time!"
         .columns.large-6
-          p You've set up automated builds and Docker container publishing. Now, let's see Habitat Builder in action!
+          p You have set up automated builds and Docker container publishing. Now, let's see Habitat Builder in action!
```
Comment (cnunciato) [inline www/source/demo/steps/8.html.slim]: `Now, let us`...? :wink:

Comment (nathenharvey) [reply www/source/demo/steps/8.html.slim]: I almost updated it to say that.  Or just deleted `Now, let's see Habitat Builder in action!` 😄

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/308678509).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/308678509).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**